lib/Zonemaster/Backend/Config.pm

```diff
@@ -425,7 +425,7 @@ Returns an integer.
 Get the value of
 L<ZONEMASTER.number_of_processes_for_frontend_testing|https://github.com/zonemaster/zonemaster-backend/blob/master/docs/Configuration.md#number_of_processes_for_frontend_testing>.
 
-Returns a positive integer.
+Returns a strictly positive integer.
```

Comment:
Some POD says `Returns an integer.`, should we update them as well ?

Reply:
The idea I'm trying to implement is to let the configuration documentation specify the range of valid values in the configuration file, and to let the POD specify the data type of the return values. This way we can communicate the important details while keeping the redundancy across files to a minimum.

Since Perl is so loosely typed it's somewhat arbitrary how precisely the data type should be specified. I took another look at this. For scalar values the [perldata/Variable names](https://perldoc.perl.org/perldata#Variable-names) seems to provide a meaningful granularity without going into too much detail. I.e. distinguishing between numbers, strings and undefined.

I'll make an update to this effect.

Reply:
Oh, and I forgot. Some properties perform a transformation of the values parsed from the config file. These transformations also need to be documented. Especially for the complex configuration properties I believe these transformations are best described in relation to the the data type returned internally by the getter methods. So I think the documentation for these transformations should also be located in the POD for the getters.